db/migrate/20170405065305_remove_image_password_audit.rb

```diff
@@ -0,0 +1,20 @@
+class RemoveImagePasswordAudit < ActiveRecord::Migration
+  def up
+    Audited::Audit.where(:auditable_type => 'Image').find_in_batches do |batch|
+      batch.each do |audit|
+        audit.audited_changes.delete("password")
```

Comment:
well I could add a condition whether there's only 'password' key but I can simply try deleting the key and checking the hash, if there are no more audit changes then we simply destroy the whole record